matchbox_socket/src/webrtc_socket/native.rs

```diff
@@ -461,8 +471,23 @@ async fn create_rtc_peer_connection(
         })
     }));
 
-    connection.on_peer_connection_state_change(Box::new(move |s| {
-        debug!("Peer Connection State has changed: {}", s);
+    connection.on_peer_connection_state_change(Box::new(move |state| {
+        debug!("Peer Connection State has changed: {state}");
+        match state {
+            // todo: double check meaning of all these states
+            RTCPeerConnectionState::Unspecified
+            | RTCPeerConnectionState::New
+            | RTCPeerConnectionState::Connecting
+            | RTCPeerConnectionState::Connected => {}
```

Comment:
Why are we not sending the events for Connecting/Connected?

Reply:
In our current API, a peer is not considered connected until all the data channel are established.

We could perhaps add a connecting state in this pr, but due to the inconsistencies between the platforms/browsers I'm wondering if it's safer to stick to really simple connected/disconnected, and then try to expand in a later pr.